lib/endpoints/class-wp-rest-users-controller.php

```diff
@@ -656,7 +656,7 @@ public function get_item_schema() {
 				'slug'        => array(
 					'description' => 'An alphanumeric identifier for the object unique to its type.',
 					'type'        => 'string',
-					'context'     => array( 'embed', 'view', 'edit' ),
+					'context'     => array( 'view', 'edit' ),
```

Comment:
@danielbachhuber It is, but it also defaults to the `user_login` so it makes sense to me to not include it in the embedded response.